Problem building because of missing references to math functions on Linux

### Bug Description

Unsure if the problem is just me or my setup, but I could use some assistance on installation if possible.

I've added the SwiftImGui package to a new Swift project according to the README. On build it seems like there's a problem linking functions from `math.h`.

### Information

- **Package version:** `1.82.0`
- **Platform version:** `Ubuntu 20.04`
- **Swift version:**
```
Swift version 5.5.1 (swift-5.5.1-RELEASE)
Target: x86_64-unknown-linux-gnu
```

### Checklist

- [x] If possible, I've reproduced the issue using the `main`/`master` branch of this package.
- [x] I've searched for existing issues under the issues tab.
- [x] The bug is reproducible

### Steps to Reproduce

1. Made a new project, by initializing it in an empty folder:
```
swift package init --type executable
```
2. Added `SwiftImGui` to `Package.swift` according to the README:
```
import PackageDescription

let package = Package(
    name: "TestProject",
    dependencies: [
        .package(url: "https://github.com/ctreffs/SwiftImGui.git", from: "1.82.0")
    ],
    targets: [
        .executableTarget(
            name: "TestProject",
            dependencies: [
                .product(name: "ImGui", package: "SwiftImGui")
            ]),
        .testTarget(
            name: "TestProjectTests",
            dependencies: ["TestProject"]),
    ]
)
```
3. Added `import ImGui` to the `main.swift` file
4. Tried building using `swift build`

### Expected behavior

Expected it to build the project.

### Actual behavior

Seems like there was a problem linking `math.h`

```error: link command failed with exit code 1 (use -v to see invocation)
.build/checkouts/SwiftImGui/Sources/CImGui/./imgui/imgui_internal.h:369: error: undefined reference to 'powf'
.build/checkouts/SwiftImGui/Sources/CImGui/./imgui/imgui_internal.h:370: error: undefined reference to 'pow'
.build/checkouts/SwiftImGui/Sources/CImGui/./imgui/imgui_internal.h:371: error: undefined reference to 'logf'
.build/checkouts/SwiftImGui/Sources/CImGui/./imgui/imgui_internal.h:372: error: undefined reference to 'log'
.build/checkouts/SwiftImGui/Sources/CImGui/./imgui/imgui_internal.h:397: error: undefined reference to 'sqrtf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui.cpp:1839: error: undefined reference to 'fmodf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_demo.cpp:430: error: undefined reference to 'fmodf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_demo.cpp:7342: error: undefined reference to 'fmodf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_demo.cpp:7344: error: undefined reference to 'fmodf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_demo.cpp:6071: error: undefined reference to 'floorf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_demo.cpp:1203: error: undefined reference to 'cosf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_demo.cpp:1203: error: undefined reference to 'sinf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_demo.cpp:1576: error: undefined reference to 'cosf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_demo.cpp:2891: error: undefined reference to 'sinf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_demo.cpp:5781: error: undefined reference to 'sqrtf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_demo.cpp:1600: error: undefined reference to 'sinf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_draw.cpp:375: error: undefined reference to 'cosf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_draw.cpp:375: error: undefined reference to 'sinf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_draw.cpp:377: error: undefined reference to 'cosf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_draw.cpp:390: error: undefined reference to 'acosf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_draw.cpp:390: error: undefined reference to 'ceilf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_draw.cpp:556: error: undefined reference to 'acosf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_draw.cpp:556: error: undefined reference to 'ceilf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_draw.cpp:740: error: undefined reference to 'sqrtf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_draw.cpp:933: error: undefined reference to 'sqrtf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_draw.cpp:1163: error: undefined reference to 'ceilf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_draw.cpp:1184: error: undefined reference to 'ceilf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_draw.cpp:3782: error: undefined reference to 'acosf'
ned reference to 'floorf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imstb_truetype.h:2641: error: undefined reference to 'floorf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imstb_truetype.h:4199: error: undefined reference to 'floorf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_widgets.cpp:5033: error: undefined reference to 'atan2f'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_internal.h:369: error: undefined reference to 'powf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_internal.h:371: error: undefined reference to 'logf'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_internal.h:372: error: undefined reference to 'log'
.build/checkouts/SwiftImGui/Sources/CImGui/imgui/imgui_internal.h:370: error: undefined reference to 'pow'
clang-10: error: linker command failed with exit code 1 (use -v to see invocation)
[0/1] Linking TestProject
```

--- a/.github/workflows/ci-linux.yml
+++ b/.github/workflows/ci-linux.yml
@@ -17,8 +17,10 @@ jobs:
       image: swift:${{ matrix.swift }}
     steps:
       - name: Checkout
-        uses: actions/checkout@master
+        uses: actions/checkout@v3.5.2
+
       - name: Test
-        run: make test
+        run: swift test -Xcxx -Wno-modules-import-nested-redundant -Xcxx -Wno-return-type-c-linkage -Xcc -Wno-modules-import-nested-redundant -Xcc -Wno-return-type-c-linkage
+      
       - name: Build Release
-        run: make build-release
+        run: swift build -c release -Xcxx -Wno-modules-import-nested-redundant -Xcxx -Wno-return-type-c-linkage -Xcc -Wno-modules-import-nested-redundant -Xcc -Wno-return-type-c-linkage

--- a/Package.swift
+++ b/Package.swift
@@ -11,7 +11,8 @@ var package = Package(
         .target(name: "CImGui",
                 path: "Sources/CImGui",
                 cSettings: [.define("CIMGUI_DEFINE_ENUMS_AND_STRUCTS")],
-                cxxSettings: [.define("CIMGUI_DEFINE_ENUMS_AND_STRUCTS")]),
+                cxxSettings: [.define("CIMGUI_DEFINE_ENUMS_AND_STRUCTS")],
+                linkerSettings: [.linkedLibrary("m", .when(platforms: [.linux]))]),
         .target(name: "AutoWrapper",
                 resources: [
                     .copy("Assets/definitions.json")

